another.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE
+  client (
+    id INTEGER,
+    name VARCHAR(100),
+    price DECIMAL(10, 5),
+    age UNSIGNED BIG INT,
+    organization_name NATIVE CHARACTER(70)
+  );
```

Comment:
This file should be lower-case, but it seems that `prettier-plugin-sql-cst` isn't picking it up.

It's definitely picking up the local prettier config:

<img width="792" alt="image" src="https://user-images.githubusercontent.com/10214025/220348326-45503fc7-e2af-4c4d-9226-5e199cb55c45.png">

Reply:
Since we are only using types from pgmeta, I've removed unnecessary dependencies imported by default during npm install. After this change, CI is now picking up sql files.